docs/reference/standard-library.rst

```diff
@@ -38,40 +38,77 @@ Int
 Sequences
 ============
 
-Required for `pluscal procedures <procedure>`.
+Required for `pluscal procedures <procedure>`. A sequence is like a list in any other language. You write it like `<<a, b, c>>`, and the elements can be any other values (including other sequences).
 
 Seq(set)
-  The set of all sequences where the values are all elements of ``set``. |noenumerate| 
+  The set of all sequences where the values are all elements of ``set``. As the output is computed in a lazy way, this is not memory/cpu intensive. |noenumerate|
+
+  ::  
+
+  Seq({a, b}) = Seq({a, b})
+  \* in practice, the output is a set that contains the values:
+  \* {<<>>, <<a>>, <<b>>, <<a, b>>, <<b, a>>}
```

Comment:
This is incorrect: it's **all** sequences made from 'a's and 'b's, so it also includes the sequence `<<a, a, a, a, b, a, b>>`. That's why it's nonenumerable.